s27125-HubertJozwiak/zajecia2/zadanie3.txt

```diff
@@ -0,0 +1,7 @@
+PATH - przechowuje ścieżki do katalogów, w których system operacyjny szuka programów do wykonania. Dzięki tej zmiennej można uruchamiać programy z dowolnego miejsca w systemie, bez konieczności podawania pełnej ścieżki do pliku wykonywalnego.
+RANDOM - przechowuje losową liczbę całkowitą. Może być przydatna w skryptach, które wymagają generowania losowych wartości.
+PWD - przechowuje bieżący katalog roboczy. Może być przydatna w skryptach, które wymagają dostępu do plików w bieżącym katalogu.
+PS1 - przechowuje znak zachęty powłoki. Może być przydatna do dostosowania wyglądu powłoki, np. dodania kolorów lub informacji o bieżącym katalogu.

```

Comment:
Oczywista oczywistość. Wysil się pomyśl jak można lepiej to wykorzystać. Ponadto ja to omawiałem na ćwiczeniach.